CONTRIBUTING.md

```diff
@@ -75,14 +80,15 @@ jake runtests tests=2dArrays debug=true
 ```
 
 ## Adding a Test
-To add a new testcase, simply place a `.ts` file in `tests\cases\compiler` containing code that exemplifies the bugfix or change you are making.
+
+To add a new test case, simply place a `.ts` file in `tests\cases\compiler` containing code that exemplifies the bugfix or change you are making.
 
 These files support metadata tags in the format  `// @metaDataName: value`. The supported names and values are:
 
-* `comments`, `sourcemap`, `noimplicitany`, `declaration`: true or false (corresponds to the compiler command-line options of the same name)
-* `target`: ES3 or ES5 (same as compiler)
-* `out`, outDir: path (same as compiler)
-* `module`: local, commonjs, or amd (local corresponds to not passing any compiler --module flag)
+* `comments`, `sourcemap`, `noimplicitany`, `declaration`: `true` or `false` (corresponds to the compiler command-line options of the same name)

```

Comment:
i think that:
all compiler options can be passed to the test harness by prefixing them with `@` e.g. `@removeComments: true` or `@outFile: a\b.js`

then leave the entry that already exits about fileName